src/Http/Authentication.Abstractions/src/AuthenticationProperties.cs

```diff
@@ -30,6 +31,7 @@ public AuthenticationProperties()
         /// Initializes a new instance of the <see cref="AuthenticationProperties"/> class.
         /// </summary>
         /// <param name="items">State values dictionary to use.</param>
+        [JsonConstructor]
```

Comment:
Why this one?

Reply:
I went with this one as the comment on the `Parameters` property said it wasn't intended to be used for serialisation/persistence.

I did consider adding `[JsonIgnore]` to that property, but adding the attribute here had the same effect.